packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx

```diff
@@ -0,0 +1,423 @@
+import { Button, Icon, Image, Text } from 'blockchain-info-components'
+import { fiatToString } from 'core/exchange/currency'
+import { FlyoutWrapper } from 'components/Flyout'
+import { FormattedMessage } from 'react-intl'
+import { Props as OwnProps, SuccessStateType } from '.'
+import { path } from 'ramda'
+
+import { UserTierType } from 'data/types'
+import { WalletFiatType } from 'core/types'
+import React from 'react'
+import styled from 'styled-components'
+
+import { ITEMS, TIER_TYPES, TIERS } from './model'
+
+import {
+  BlueCartridge,
+  ErrorCartridge,
+  OrangeCartridge,
+  SuccessCartridge
+} from 'components/Cartridge'
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+`
+const Title = styled(Text)`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 7px;
+`
+const IconsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 100%;
+`
+const ContentItem = styled.div<{ isClickable?: boolean }>`
+  border-top: 1px solid ${props => props.theme.grey000};
+  padding: 20px 40px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  cursor: ${props => (props.isClickable ? 'pointer' : 'auto')};
+`
+const IconWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+const TierDescription = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0 10px;
+`
+const FooterWrapper = styled(FlyoutWrapper)`
+  flex: 1;
+  justify-content: flex-end;
+  display: flex;
+  flex-direction: column;
+`
+const CartridgeWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  justify-content: flex-end;
+`
+
+const Success = styled(SuccessCartridge)`
+  font-size: 12px;
+`
+const Errors = styled(ErrorCartridge)`
+  font-size: 12px;
+`
+const Blue = styled(BlueCartridge)`
+  font-size: 12px;
+`
+const Orange = styled(OrangeCartridge)`
+  font-size: 12px;
+`
```

Comment:
please modify the base cartridge to take a font-size prop and pass the size in that way

Reply:
Hi